frontend/admin/src/js/components/PoolDashboard.tsx

```diff
@@ -46,6 +47,12 @@ const routes = (
       redirect: loggedIn ? paths.poolTable() : undefined,
```

Comment:
Hey, we can change this to redirect to the dashboard now, like it was supposed to all along!
```suggestion
      redirect: loggedIn ? paths.dashboard() : undefined,
```